rubix-hadoop2/src/test/java/com/qubole/rubix/hadoop2/TestHadoop2ClusterManager.java

```diff
@@ -205,8 +205,8 @@ public void testClusterIndex()
         TestHadoop2ClusterManagerUtil.CLUSTER_NODES_ENDPOINT,
         worker.new MultipleWorkersOneNew(), conf, ClusterType.HADOOP2_CLUSTER_MANAGER);
     ClusterManager manager = TestHadoop2ClusterManagerUtil.getClusterManagerInstance(ClusterType.HADOOP2_CLUSTER_MANAGER, conf);
-    int index = manager.getNodeIndex(nodeHostnames.size(), "1");
+    //int index = nodeHostnames.indexOf(manager.getNode(nodeHostnames.size(), "1").getKey());
```

Comment:
this test can't be run with this change, as we need to insert nodes in the consistent hash ring.

Reply:
we can change it test for nodeName

Reply:
changed it to return nodename and verify.